src/lib.rs

```diff
@@ -1056,6 +1063,12 @@ impl Config {
         self
     }
 
+    /// Specify vcpkg installed directory. This is useful for manifest mode and custom vcpkg installation.
```

Comment:
A little bit more explanation here would be nice, as well as a mention that the default is `$VCPKG_ROOT/installed`.

Reply:
Added below